### PR TITLE
Vanover rates

### PR DESCRIPTION
**Pull request by limonspb**
#$ option provides also actual rates close analogue.
image

### CODE REVIEW

**Review by sugaarK — APPROVED** [review]

File: presets/4.3/rates/Vanover.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/96
+#$ INCLUDE: presets/4.3/rates/defaults.txt
+
+set rates_type = BETAFLIGHT
+set roll_rc_rate = 100
+set pitch_rc_rate = 100
+set yaw_rc_rate = 100
+set roll_expo = 1
+set pitch_expo = 1
+set yaw_expo = 1
```
Comment (DusKing1) [inline presets/4.3/rates/Vanover.txt:19]: This is 10

Comment (limonspb) [reply presets/4.3/rates/Vanover.txt:19]: it is not

Comment (DusKing1) [reply presets/4.3/rates/Vanover.txt:19]: https://rotorriot.com/products/vanovers-rates
image

Comment (limonspb) [reply presets/4.3/rates/Vanover.txt:19]: I will double-check, but that's what he sent to me.

Comment (ctzsnooze) [reply presets/4.3/rates/Vanover.txt:19]: At least *someone* flies with *my* standard race centre sensitivity :-)

Comment (limonspb) [reply presets/4.3/rates/Vanover.txt:19]: Confirmed with Vanover, rotor rioir rates are not correct.

**Pull request merged**